### PR TITLE
chore(NX-3493): replace Zendesk with Salesforce links

### PR DESCRIPTION
**Pull request by tam-kis**
This PR resolves [NX-3493](https://artsyproduct.atlassian.net/browse/NX-3493)

### Description

Replaces Zendesk links with Salesforce links.

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[NX-3493]: https://artsyproduct.atlassian.net/browse/NX-3493?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by araujobarret — APPROVED** [review]

**Review by lidimayra — APPROVED** [review]
Thank you for addressing this!! ❤️

**Comment by lidimayra** [comment]
_same as https://github.com/artsy/force/pull/12274#issuecomment-1515902955_

2 of the links added in this PR are not working as expected. These URLs are matching the ones we have in the [spreadsheet](https://docs.google.com/spreadsheets/d/1Yofy_cExeEgdV1-o8rzMrGnJ6In4c0aotV5dzowhYUU/edit#gid=571260299). I'll share this on [#zendesk-sf-migration](https://artsy.slack.com/archives/C04U9SJ5A0K) channel

- https://support.artsy.net/s/article/What-is-cascade-and-popcorn-bidding-and-how-does-it-work
- https://support.artsy.net/s/article/What-Counts-as-an-Artworks-Proof-of-Authenticity
- https://support.artsy.net/s/topic/s/topic/0TO3b000000UevEGAS/contacting-a-gallery ❌
- https://support.artsy.net/s/article/What-items-do-you-accept
- https://support.artsy.net/s/topic/s/topic/0TO3b000000UevOGAS/sell-with-artsy ❌
- https://support.artsy.net/s/topic/0TO3b000000UesxGAC/sell
- https://support.artsy.net/s/article/Im-an-artist-Can-I-submit-my-own-work-to-sell

File: src/app/Scenes/Inbox/Components/Conversations/Support.tsx [hunk 1]
```diff
@@ -10,9 +9,12 @@ export const Support = () => {
       </Text>
       <Touchable
         onPress={() => {
-          navigate("https://support.artsy.net/hc/en-us/sections/360008203054-Contact-a-gallery", {
-            modal: true,
-          })
+          navigate(
+            "https://support.artsy.net/s/topic/s/topic/0TO3b000000UevEGAS/contacting-a-gallery",
```
Comment (lidimayra) [inline src/app/Scenes/Inbox/Components/Conversations/Support.tsx:13]:
```suggestion
            "https://support.artsy.net/s/topic/0TO3b000000UevEGAS/contacting-a-gallery",
```

File: src/app/Scenes/MyCollection/Screens/Artwork/MyCollectionSellingWithartsyFAQ.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 import { navigate } from "app/system/navigation/navigate"
+import { useScreenDimensions } from "app/utils/hooks"
 import { sendEmailWithMailTo } from "app/utils/sendEmail"
 import { ScrollView } from "react-native"
-import { useScreenDimensions } from "app/utils/hooks"
 
 export const MyCollectionSellingWithArtsyFAQ = () => {
   const { safeAreaInsets } = useScreenDimensions()
-  const article = "https://support.artsy.net/hc/en-us/sections/360008311913-Sell-with-Artsy"
+  const article = "https://support.artsy.net/s/topic/s/topic/0TO3b000000UevOGAS/sell-with-artsy"
```
Comment (lidimayra) [inline src/app/Scenes/MyCollection/Screens/Artwork/MyCollectionSellingWithartsyFAQ.tsx:9]:
```suggestion
  const article = "https://support.artsy.net/s/topic/0TO3b000000UevOGAS/sell-with-artsy"
```

**Review by lidimayra — CHANGES_REQUESTED** [review]

File: src/app/Scenes/MyCollection/Screens/Artwork/MyCollectionSellingWithartsyFAQ.tsx (hunk 2 above)
Comment (tam-kis) [reply src/app/Scenes/MyCollection/Screens/Artwork/MyCollectionSellingWithartsyFAQ.tsx]: Thanks for noting the wrong links from the sheet, I updated it.

**Review by lidimayra — APPROVED** [review]

**Review by MrSltun — APPROVED** [review]
🚀 ✨

**Pull request merged**